src/test/resources/Hotels/hotelsproject.feature

```diff
@@ -0,0 +1,70 @@
+Feature: HotelsProject
```

Comment:
Try to add @tag at Feature file and Scenario level and use those to run tests from TestRunner.